test/IDE_tests.jl

```diff
@@ -204,3 +204,35 @@ w_real  = [1/x^2 for x in xs]
 # plot!(xs,u_predict)
 # plot(xs,w_real)
 # plot!(xs,w_predict)
+
+println("Indefinite Integral Test")
```

Comment:
Probably you mean "Infinity Integral Test"

Reply:
Good catch. I think the proper term is "Improper integral"